app/services/users/delete_articles.rb

```diff
@@ -3,7 +3,7 @@ module DeleteArticles
     module_function
 
     def call(user, cache_buster = CacheBuster)
-      return unless user.articles.any?
+      return if user.articles.blank?
```

Comment:
as we iterate with `.map` right after (which issues a select * anyway), this is going to issue one less query.